external/webserver/http_client.c

```diff
@@ -1093,13 +1098,13 @@ static int http_send_buffer(struct http_client_t *client, const char *buf, int l
 	return 0;
 }
 
-int http_send_response_helper(struct http_client_t *client, int status, const char* status_message, const char* body, struct http_keyvalue_list_t *headers)
+int http_send_response_helper(struct http_client_t *client, int status, const char* status_message,
+				const char* body, int body_len, struct http_keyvalue_list_t *headers)
```

Comment:
This multi-line function definition can be taken care of in one line itself by using separators please.

Reply:
not required